mkdocs.yml

```
@@ -58,6 +58,7 @@ nav:
   - operator-manual/custom-styles.md
   - operator-manual/ui-customization.md
   - operator-manual/metrics.md
+  - operator-manual/pprof.md
```

I would call this "Profiling". It can be expanded later if other tooling is added. 
Also not everyone knows what `pprof` is, so I think it would be clearer if it were "Profiling"
```suggestion
  - operator-manual/profiling.md
```